config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -353,7 +350,7 @@
 			description: [
 				"Alloys will be needed commonly for further progression and without a little extra effort, they can become quite costly to make. Enter the Enrichment Chamber. "
 				""
-				"Running Ores through the Enrichment Chamber will double the yield from each ore. "
+				"Running Raw Ores through the Enrichment Chamber will increase the yield from each ore to 1.33. Ore doubling is still a thing here, but you need to process full ore blocks for that."
```

Comment:
`still a thing here` suggests the player knows what came before. Making relative statements to prior versions isn't really helpful as many won't have that knowledge. So I'd just avoid it outright. 

`Running Raw Ores through the Enrichment Chamber will increase the yield by 33 percent, while running Full Ore Blocks will double the output`